src/components/MediatedRequestsActivities/MediatedRequestsActivities.test.js

```diff
@@ -47,19 +83,15 @@ describe('MediatedRequestsActivities', () => {
     expect(screen.getByText(labelIds.paneTitle)).toBeVisible();
   });
 
-  it('should trigger NavigationMenu with correct props', () => {
+  it('should render navigation menu with correct props', () => {
```

Comment:
I think trigger was better because in the test you check component call, not component render.
And could you check other files?

Reply:
Fixed